app/models/user.rb

```diff
@@ -43,6 +44,6 @@ def leader_of
 
   validates :email, presence: true
 
-  has_attached_file :photo, :styles => { :medium => "500x500>", :thumb => "150x150#" }, :default_url => "/images/:style/missing.png"
+  has_attached_file :photo, :styles => { :medium => "500x500>", :thumb => "150x150#" }, default_url: "thumb/default.png"
```

Comment:
Just curious, @shannonchu, why are there both placeholder user images? Don't we only need the larger of the two and ImageMagick automatically generates a thumbnail for us?